<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogBookingAdd</class>
 <widget class="QDialog" name="DialogBookingAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add New Booking</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>281</width>
     <height>174</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Room</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Check-out</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QDateEdit" name="txtCheckOut">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="txtGuestName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Guests Number</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="txtGuestsNumber"/>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="cmbRooms"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Check-in</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QDateEdit" name="txtCheckIn">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Guest Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>351</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="btnAccept">
      <property name="text">
       <string>Accept</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnRoomInfo">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Info</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
